tests/integration/test_multi.py

```diff
@@ -2,28 +2,33 @@
 Test suite for functions that call other functions
 """
 
-from types import FunctionType
-from typing import Iterable, Union
-
 import pytest
 from parameterized import parameterized
 
 from .base_tests import BaseTestWrapper
 from .functions import *
-from .utils import name_func
+from .utils import Parameters, name_func
 
 PARAMETERS = [
-    ([multi_funcs, multiplier, p2vrange], [(13, 17)]),
-    ([fib_product, multiplier, fib, p2vrange], [10, 20]),
-    ([fib_product, multiplier, fib, p2vrange], [10, 20]),
-    ([fib, p2vrange], range(10, 31, 10)),
-    ([quad_multiply, multiplier_generator], [(3, 7), (31, 43)]),
-    ([double_for, p2vrange], [5, 10, 15, 20]),
-    ([dupe, p2vrange], [(0, 10, 1), (3, 73, 7)]),
-    ([olympic_logo_naive, circle_lines], [(10, 10, 4), (13, 13, 7)]),
-    (
-        [olympic_logo, olympic_logo_mids, circle_lines],
-        [(10, 10, 4), (13, 13, 7)],
+    Parameters(func=multi_funcs, helpers=[multiplier, p2vrange], args_list=[(13, 17)]),
+    Parameters(
+        func=fib_product, helpers=[multiplier, fib, p2vrange], args_list=[10, 20]
+    ),
+    Parameters(func=fib, helpers=[p2vrange], args_list=range(10, 31, 10)),
+    Parameters(
+        func=quad_multiply, helpers=[multiplier_generator], args_list=[(3, 7), (31, 43)]
+    ),
+    Parameters(func=double_for, helpers=[p2vrange], args_list=[5, 10, 15, 20]),
+    Parameters(func=dupe, helpers=[p2vrange], args_list=[(0, 10, 1), (3, 73, 7)]),
+    Parameters(
+        func=olympic_logo_naive,
+        helpers=[circle_lines],
+        args_list=[(10, 10, 4), (13, 13, 7)],
+    ),
+    Parameters(
+        func=olympic_logo,
```

Comment:
Make consistent

Reply:
Unfortunately, I think that's a limitation of the auto-formatter I'm using.

Reply:
Iirc, auto-formatters only look at one line at a time. If a line is too long, it will shorten it into multiple ones. My suggestions to make the formatting look more consistent is to make all Parameter's be multi-line

Reply:
Okay yea that works!